next.config.js

```diff
@@ -1,16 +1,15 @@
+/* eslint-disable no-undef */
+
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   rewrites: async () => {
     return [
       {
         source: '/api/:path*',
-        destination:
-          process.env.NODE_ENV === 'development'
-            ? 'http://127.0.0.1:5328/api/:path*'
-            : '/api/',
-      },
-    ]
-  },
-}
+        destination: process.env.NODE_ENV === 'development' ? 'http://127.0.0.1:5328/api/:path*' : '/api/'
```

Comment:
There's some formatting here (and in a couple of other places I'll note) where this is now one line where my local prettier had been breaking into multiple lines. I prefer the multiple, especially since I have my column width set pretty low and these end up wrapping weird

Would you be open to getting that formatting back in? Any clue how, I was looking but didn't see it?

Reply:
totally minor though, we can get it figured later or not!

Reply:
I had changed prettier`s line wrapping threshold up to 140. I changed it back now, and reformatted. All the examples you mentioned are now back to wrapping.